client/my-sites/media-library/header.jsx

```diff
@@ -123,6 +124,14 @@ export default React.createClass( {
 			<header className="media-library__header">
 				<h2 className="media-library__heading">{ this.translate( 'Media Library' ) }</h2>
 				{ this.renderUploadButtons() }
+				<MediaModalSecondaryActions
+					selectedItems={ this.props.selectedItems }
+					onViewDetails={ this.props.onViewDetails }
+					onDelete={ this.props.onDeleteItem }
+					renderStorage={ false }
+					site={ this.props.site }
+					view={ 'LIST' }
+				/>
```

Comment:
A trivial detail, but sometimes you're closing the component instance on a new line and sometimes not.